assembl/static2/js/app/components/common/modal.jsx

```diff
@@ -75,7 +76,7 @@ class AssemblModal extends React.Component<Props, State> {
 
     return (
       <Modal bsSize={bsSize || null} show={this.state.showModal} onHide={this.close}>
-        <Modal.Header closeButton>{title && <Modal.Title>{title}</Modal.Title>}</Modal.Header>
+        <Modal.Header>{title && <Modal.Title>{title}</Modal.Title>}</Modal.Header>
```

Comment:
Why did you remove `closeButton`?

Reply:
Oups, that's a nice mistake. I just wanted to remove the withClosingCross prop that Felix had added